src/display/editor/annotation_editor_layer.js

```diff
@@ -794,10 +792,9 @@ class AnnotationEditorLayer {
   }
 
   createAnnotationNode() {
-    this.removeTempHighlight();
-    const { event, isCentered, ...highlightParams } = this.#highlightNodeParams;
-    this.#createAndAddNewEditor(event, isCentered, highlightParams);
-    this.#highlightNodeParams = null;
+    const { event, data } = this.#highlightSelectionParams;
+    this.#createAndAddNewEditor(event, false, data);
+    this.#highlightSelectionParams = null;
```

Comment:
Remove this as this is redundant. This variable is always reset in `removeTempHighlight` which is guaranteed to call after `createAnnotationNode`

Reply:
Oh sorry bro.  I will remove